src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -235,18 +234,38 @@ export default class BrowserCell extends Component {
       content = <span>&nbsp;</span>;
       classes.push(styles.empty);
     } else if (type === 'Pointer') {
+      const defaultPointerKey = await localStorage.getItem(value.className) || 'objectId';
+
+      let dataValue = value.id;
+      if ( defaultPointerKey !== 'objectId' ) {
+        dataValue = value.get(defaultPointerKey);
+        if ( dataValue && typeof dataValue === 'object' ){
```

Comment:
We need something else than `Invalid Value` in case the pointer key is set to a field that is `(undefined)`.

<img width="154" alt="image" src="https://user-images.githubusercontent.com/5673677/120922603-654a6580-c6ca-11eb-8039-a24371b10a0e.png">

I suggest to display `(undefined)` which is the true current value of the pointer key, or maybe you have a different idea.

Reply:
I can replace the square brace but from my understanding of the dashboard, the undefined is not clickable. The idea that I had while implementing this was that if there's an issue while loading the pointer key value, display something but still leave the field clickable to allow users to navigate the between the pointer relation.
While undefined indicates that there is no or the row doesn't point to any row in the respective table.

Reply:
The word "invalid" has a different meaning than "undefined". If a pointer key value is "undefined", we should convey that it is simply "undefined". But we also cannot display it the same as `(undefined)`, which means that no value is set for the field.

Since we will abandon the pill design with https://github.com/parse-community/parse-dashboard/pull/1708 I think we end up with these ambiguations:

- a) `(undefined)` = no value is set for field
- b) `undefined ↗️` = pointer field value is defined as string "undefined"
- c) `(undefined) ↗️` = pointer field value is not defined

This means that a pointer key value that is undefined can only be displayed as (c).

Reply:
Ok, I'll update it to match the PR branch #1708 and update this PR to match (c)

Reply:
I think we should merge the pointer redesign PR first, then merge master into this PR?

Reply:
I have also updated the PR with the new pointer design.